code/src/components/content/ContentContainer.js

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable operator-linebreak */
+/* eslint-disable indent */
+import React, { useRef } from 'react';
+import styled from 'styled-components/macro';
+import { useSelector } from 'react-redux';
+import { textColor } from 'components/GlobalStyles';
+import TaskList from './TaskList';
+import Cat from './Cat';
+import NewTask from './NewTask';
+import DeleteAll from './DeleteAll';
+
+const ContentContainer = () => {
+  const taskListLength = useSelector((store) => store.tasks.tasks.length);
+  const headerText =
+    taskListLength === 0
+      ? 'You have no tasks'
+      : `You have ${taskListLength} ${
+          taskListLength === 1 ? 'task' : 'tasks'
+        } `;
```

Comment:
I wanted to do something like this before but didn't figure out how. But now I know. Thanks!